cmd/oceanbench/set.go

```diff
@@ -374,6 +376,99 @@ func editDevicesHandler(w http.ResponseWriter, r *http.Request) {
 	http.Redirect(w, r, "/set/devices?ma="+ma, http.StatusFound)
 }
 
+// configDevicesHandler handles configuration of new devices.
+//
+// Form Fields:
+//
+//	name = Name of the new device
+//	ma = MAC address
```

Comment:
For consistency, please use standard two-letter query params, i.e., `ma` for MAC address, `ct` for client type, `wi` for the command separated string ssid-name, ssid-password, `ll` for comma-separated latitude,longitude, etc.

Reply:
PS These query params form part of our REST API, that is why it's important to keep them consistent.

Reply:
done